packages/xcoobee-cookie-kit-core/src/LocaleManager.js

```diff
@@ -11,7 +18,10 @@ export function clearCountryCode() {
 }
 
 export function getLocale() {
-  const locale = localStorage[localeKey];
+  let locale = localStorage[localeKey];
+  // We used to store "DE", "EN", "ES", and "FR" for locales. Convert to expected
```

Comment:
For some reason we were saving the "display text" of the locale instead of a complete language code with locale which is what we expect in `textMessage`. I think it is better to save the complete language code instead.

To be backward compatible, this code converts the display text to a language code.

This change will make it easier to support more the four default languages (en-us, de-de, es-419, and fr-fr).

Reply:
Dan, we already should have a mapping for LOCALE_TO_LOCALE in 
https://github.com/XcooBee/xcoobee-cookie-kit/blob/develop/packages/xcoobee-cookie-kit-core/src/countryData.js

Ok. I see that use countryMapping later on.